src/jdk.unsupported/share/classes/sun/misc/Unsafe.java

```diff
@@ -997,6 +979,27 @@ public int arrayBaseOffset(Class<?> arrayClass) {
         return (int) theInternalUnsafe.arrayBaseOffset(arrayClass);
     }
 
+    @ForceInline
+    private static void assertNotTrusted(Field f) {
```

Comment:
I don't think this can be named assertXXX, needs to something like ensureNotTrusted.